contrib/opencensus-ext-stackdriver/opencensus/ext/stackdriver/stats_exporter/__init__.py

```diff
@@ -402,21 +363,34 @@ def get_user_agent_slug():
     return "opencensus-python/{}".format(__version__)
```

Comment:
Curious (although it is not added by this PR) - which version do we want here (opencensus or ext package or both)?

Reply:
The other clients report the core library version, but I think there's a good argument for replacing this with just the ext version.